presto-spi/src/main/java/com/facebook/presto/spi/block/IntArrayAllocator.java

```diff
@@ -11,21 +11,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package com.facebook.presto.spi.block;
 
 import java.util.ArrayList;
-import java.util.List;
 
 public class IntArrayAllocator
 {
-    private List<int[]> arrays = new ArrayList();
+    ArrayList arrays = new ArrayList();
 
     public int[] getIntArray(int size)
```

Comment:
This method is not used

Reply:
Actually, none of the methods of this class are being used.

Reply:
It's not used for this code path.  When I switch branches it will go away entirely.